Header header
float64 tau
std_msgs/Float64MultiArray x
std_msgs/Float64MultiArray e
std_msgs/Float64MultiArray e_dot
std_msgs/Float64MultiArray z
std_msgs/Float64MultiArray z_dot
std_msgs/Float64MultiArray f